text/0000-deprecate-ember-logger.md

```diff
@@ -0,0 +1,265 @@
+calling Ember.Logging and should probably be given an early heads-up to 
+adjust their code to use console in advance of the release containing the 
+deprecation to limit the level of pain that their users experience. In 
+the order of their reliance on Ember.Logging:
+
+* ember-concurrency (15)
+* ember-cli-deprecation-workflow (9)
+* ember-stripe-service (9)
+* semantic-ui-ember (7)
+* ember-resolver (6)
+* ember-cli-page-object (4) 
+* ember-cli-sentry (3)
+* ember-islands (3)
+* ember-states (3)
+* ember-cli-pagination (2)
+* ember-cli-clipboard (1)
+* ember-cli-fastboot (1)
+* ember-elsewhere (1)
+* ember-i18n (1)
+* ember-simple-auth-token (1)
+* ember-svg-jar (1)
+* liquid-fire (1)
+
+### Timeline
+
+1. Remove all uses of Ember.Logging from other Ember and ember-data code.
+This is particularly important for any methods involved in reporting 
+deprecation. [Date? This could start now.]
+2. Write and test codemod to change code to use console and make it available. [Any time]
+3. Inform users that Ember.Logging will be deprecated and in what release. [When we know schedule]
+4. Call deprecation APIs from all Logging methods in @ember-console/index.js [Once codemod is out there]
+5. In documentation, offer suggestions for babel plugins available to suppress [Before we release]
+console calls in production builds with any special instructions for how to 
+use them in ember-cli broccoli builds.
+6. In due season, the deprecation will appear in a release. [When?]
+
+## How we teach this
+
+### Official code bases and documentation
+
+We do not currently actively teach the use of Ember.Logger. We will need to 
+remove any passing references to Ember.Logger from the Ember guides 
+and from the Super Rentals tutorial, if any such references exist. 
+
+Once it is gone from the code, we need to verify it no longer appears in the 
+API listings.
+
+Likewise, we will need to remove the use of Ember.Logger from ember, ember-data, 
+and the website.
+
+## Drawbacks
+
+191 projects in Ember Inspector are using it. It has been there and documented 
+for a long time. So this deprecation will cause API churn on a lot of projects.
+Hopefully providing a codemod to replace Ember.Logger calls with the 
+corresponding console calls will make this transition relatively painless.
+
+However, those using Logging.debug as something different from Logging.log
+may beg to differ, even though under the covers Logging.debug only calls 
+console.debug if it exists, calling console.log otherwise.
+
+## Alternatives
+
+1. Leave things as they are, perhaps providing @ember/console module interface.
+
+2. Extract Ember.Logging into its own (tiny) @ember/logging package as a shim for users.
+
+## Unresolved questions
+
+What mechanisms are available with Babel to suppress console calls in production builds? 
+Which if any would we point users toward?
+
+How do I write a codemod? I may need help on this part.
```

Comment:
I wouldn't say this is an unresolved question. You will definitely get some help here :)